param/__init__.py

```diff
@@ -1154,10 +1170,10 @@ class Composite(Parameter):
 
     __slots__ = ['attribs', 'objtype']
 
-    def __init__(self, attribs=None, **kw):
-        if attribs is None:
+    def __init__(self, attribs=Undefined, **kw):
+        if attribs is Undefined:
             attribs = []
-        super(Composite, self).__init__(default=None, **kw)
+        super(Composite, self).__init__(default=Undefined, **kw)
```

Comment:
Is there a reason that `attribs` is not handled using `_slot_defaults` like other arguments?